src/fileActions.jsx

```diff
@@ -625,3 +625,31 @@ export const updateFile = (file, currentPath) => {
                     return file;
             }, exc => console.error(`Adding file ${file} failed: ${exc.toString()}`));
 };
+
+export const downloadFile = (currentPath, selected) => {
+    const query = window.btoa(JSON.stringify({
+        payload: "fsread1",
+        binary: "raw",
+        path: `${currentPath}/${selected.name}`,
+        superuser: "try",
+        // TODO: is this a good value?
+        max_read_size: 150 * 1024 * 1024,
```

Comment:
It's a bit arbitrary. There is little reason why this shouldn't work for bigger files, like VM images. This has to be chunked anyway, "all in RAM/single read" files would have to be way smaller than 150 MB, more like 1 MB.

Reply:
Right, should it be equivalent to the chunk size?